src/crypto/webcrypto.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+module.exports = function getWebCrypto () {
+  const WebCrypto = require('node-webcrypto-ossl')
```

Comment:
```
WARNING
At this time this solution should be considered suitable for research and experimentation, further code and security review is needed before utilization in a production application.
```

This makes the statement that 'using webcrypto is safer' invalid because we know have to trust this shim